docs/sources/release-notes/v2.5.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Grafana Mimir version 2.5 release notes"
+menuTitle: "V2.5 release notes"
+description: "Release notes for Grafana Mimir version 2.5"
+weight: 300
+---
+
+# Grafana Mimir version 2.5 release notes
+
+Grafana Labs is excited to announce version 2.5 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Alertmanager Discord support**
+  The alertmanager can now be configured to send alerts into Discord channels.
+
+- **Configurable TLS minimum version and cipher suites**
+  We added the two new flags `-server.tls-min-version` and `-server.tls-cipher-suites`, they can be used to define the minimum version and the supported cipher suites in all HTTP and gRPC servers in Mimir.
```

Comment:
```suggestion
  We added the two new flags `-server.tls-min-version` and `-server.tls-cipher-suites`, they can be used to define the minimum TLS version and the supported cipher suites in all HTTP and gRPC servers in Mimir.
```

Reply:
Ursula, I added your suggestion manually in: e1df40df0